Exam Labs/2018-01-17 Stairs/lab/as1r2/etc/frr/frr.conf

```diff
@@ -0,0 +1,41 @@
+password zebra
+enable password zebra
+!
+log file /var/log/frr/frr.log
+
+router rip
+network 1.0.0.0/24
+redistribute connected
+redistribute bgp
+
+!
+!  BGP CONFIGURATION
+!
+debug bgp keepalives
+debug bgp updates in
+debug bgp updates out
+!
+router bgp 1
+no bgp ebgp-requires-policy
+no bgp network import-check 
+!
+! NEIGHBORS
+!
+neighbor 10.0.0.6 remote-as 2
+neighbor 10.0.0.6 description Router as2r2
+neighbor 1.0.0.1 remote-as 1
+neighbor 1.0.0.1 description Router as1r1 (iBGP)
+!
+! NETWORKS
+!
+network 1.0.0.0/24
+network 10.0.0.4/30
+
+neighbor 10.0.0.6 route-map metricOut out
+
+route-map metricOut permit 10
+set metric 10
```

Comment:
Please, add indentation to improve readability. (you can find the correct indentation running the command show running-config inside vtysh)